src/features/berthSwitchOffer/berthSwitchOfferPage.scss

```diff
@@ -0,0 +1,56 @@
+@import 'styles/mixin';
+@import 'helsinki/colors';
+@import 'helsinki/fonts';
+@import 'styles/variables';
+
+.vene-offer-page {
+  background-color: $hel-light;
+  flex-grow: 1;
+
+  &__header {
+    background-color: $hel-fog;
+
+    h2 {
+      margin-bottom: $spacing-02;
+    }
+
+    p {
```

Comment:
IMO, it's safer to avoid targeting tagnames, especially that we're not using modules.